gcp/live/dev/k8s/gpii/couchdb/terraform.tfvars

```diff
@@ -31,3 +31,4 @@ pv_reclaim_policy    = "Delete"
 pv_storage_class     = ""
 pv_provisioner       = ""
 execute_destroy_pvcs = "true"
+execute_recover_pvcs = "false"
```

Comment:
This change means that we will almost never test the `execute_recover_pvcs` workflow. Does that concern you?

Reply:
Good question. We recreate staging cluster from time to time, so recover code is tested at that time, I guess this is better than never.